layout/components/versioninfo.xml

```diff
@@ -0,0 +1,16 @@
+<root>
+	<styles>
+		<include src="file://{resources}/styles/components/versioninfo.sass" />
+	</styles>
+	<scripts>
+		<include src="file://{resources}/scripts/components/versioninfo.js" />
+	</scripts>
+
+	<Panel class="VersionInfo" onload="VersionInfo.onLoad()">
```

Comment:
can just say `update` here and get rid of `onLoad`

Reply:
Can do. I'll sort it out when I get home.